src/overlays/actors/ovl_Obj_Boyo/z_obj_boyo.c

```diff
@@ -126,7 +126,7 @@ void ObjBoyo_Update(Actor* thisx, GlobalContext* globalCtx2) {
     Actor* target;
     s32 num;
 
-    target = ObjBoyo_GetCollidedActor(thisx, globalCtx, &num);
+    target = ObjBoyo_GetCollidedActor((ObjBoyo*)thisx, globalCtx, &num);
```

Comment:
```suggestion
    target = ObjBoyo_GetCollidedActor(this, globalCtx, &num);
```

Reply:
Oh weird, I thought I specifically tried this and it didn't match... but it matches now!